contracts/StablecoinConverter.sol

```diff
@@ -503,9 +500,14 @@ contract StablecoinConverter is EpochTokenLocker {
             getRemainingAmount(order),
             getBalance(user, tokenIdToAddressMap(order.sellToken))
         );
-        uint256 limitTerm = currentPrices[order.sellToken].mul(order.priceDenominator)
-            .sub(currentPrices[order.buyToken].mul(order.priceNumerator));
-        return leftoverSellAmount.mul(limitTerm).div(order.priceDenominator);
+        uint256 limitTermLeft = currentPrices[order.sellToken].mul(order.priceDenominator);
```

Comment:
Maybe we should call them subtrahend/minuend instead of left/right?

Reply:
I would be in favour of this if we could come up with much better names than suggested here.